src/cim/components/Hypershift/types.ts

```diff
@@ -1,12 +1,32 @@
 import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
 
+export type Condition = {
```

Comment:
There is already a type for Condition in src/cim/types/k8s/shared.ts

Reply:
Fixed